dts/arm/cypress/psoc6_cm0.dtsi

```diff
@@ -15,6 +15,307 @@
 
 		/delete-node/ cpu@1;
 	};
+
+	soc {
+		intmux: intmux@40210020 {
+			/* see cypress,psoc6-int-mux.yaml */
+			compatible = "cypress,psoc6-intmux";
+			reg = <0x40210020 0x20>;
+			label = "INTMUX";
+			status = "disabled";
```

Comment:
should probably remove `status="disabled"` here.

Reply:
yes, it will be easy for user - switch to "okay"